apps/central-scan/backend/src/store.ts

```diff
@@ -982,10 +980,12 @@ export class Store {
       backInterpretationJson: string;
       frontNormalizedFilename: string;
       backNormalizedFilename: string;
+      batchSequenceId: number;
     }>) {
       yield {
         id: row.id,
         batchId: row.batchId,
+        batchSequenceId: row.batchSequenceId,
```

Comment:
Can we call this something more clear like `indexInBatch` in the code (even if we have to use `BatchSequenceId` in the CDF CVR)?

Reply:
Sure!